.github/workflows/system-tests-k8s.yml

```diff
@@ -80,6 +78,8 @@ jobs:
       - name: Run System Tests on K8s
         id: bazel-test-all
         uses: ./.github/actions/bazel-test-all/
+        env:
```

Comment:
It's the other way around. We don't publish any artifacts so we don't need AWS credentials but good to leave docker-login logic as container images in docker rules, do get pulled.